conans/client/graph/graph.py

```diff
@@ -224,7 +238,8 @@ def serialize(self):
         result.update(self.conanfile.serialize())
         result["context"] = self.context
         result["test"] = self.test
-        result["requires"] = {n.id: n.ref.repr_notime() for n in self.neighbors()}
+        result["transitive_deps"] = {d.node.id: d.require.serialize()
```

Comment:
Maybe we want to call it ``requires``? ``transitive_requires``? ``dependencies``?

Reply:
Yeah, I wanted to bring it here to discuss the best name. Perhaps, keeping `requires` is good enough? wdyt @czoido @RubenRBS ?

Reply:
I'd say `dependencies`, for symmetry with `self.dependencies` in a ConanFile

Reply:
Yes, ``dependencies`` could be the best, it is the closest, it is how ``dependencies`` is computed:
```python
 def from_node(node):
        d = OrderedDict((require, ConanFileInterface(transitive.node.conanfile))
                        for require, transitive in node.transitive_deps.items())
        return ConanFileDependencies(d)
```
It is basically a thin adaptation over ``transitive_deps``, with an interface to protect internal data on top of it (we might want to make accessible some of this data over the ``ConanFileInterface`` if necessary and requested by users)